src/ObjCRuntime/Registrar.cs

```diff
@@ -186,7 +186,7 @@ public void VerifyRegisterAttribute (ref List<Exception> exceptions)
 				}
 			}
 
-			static char[] invalidSelectorCharacters = new char[] { ' ', '\t' };
+			static char[] invalidSelectorCharacters = new char[] { ' ', '\t', '?', '\\', '!', '|', '@', '"', '\'', '%', '&', '/', '(', ')', '=', '^', '[', ']', '{', '}', ',', '.', ';', '-', '\n' };
```

Comment:
@spouliot @chamons I've added a comment to the source code about the duplicated information.